src/libs/ReportUtils.js

```diff
@@ -660,6 +660,17 @@ function hasSingleParticipant(report) {
     return report && report.participantAccountIDs && report.participantAccountIDs.length === 1;
 }
 
+/**
+ * Checks whether all the transactions linked to the IOU report are of the Distance Request type
+ *
+ * @param {string|null} iouReportID
+ * @returns {boolean}
+ */
+function hasOnlyDistanceRequestTransactions(iouReportID) {
+    const allTransactions = TransactionUtils.getAllReportTransactions(iouReportID);
+    return _.all(allTransactions, (transaction) => TransactionUtils.isDistanceRequest(transaction));
```

Comment:
I can't find the link or doc, but I thought we're going to prioritize using the built-in methods over lodash if they exist. We could use 

```
    return allTransactions.every(transaction => TransactionUtils.isDistanceRequest(transaction));
```

@Julesssss Can you confirm this?

Reply:
As far as I understand, this is more related to TypeScript, as it's more fail-safe. But if you insist, I can change to `.every` here

Reply:
I am also not 100% sure on this one.

Reply:
Then I would like to keep it as is until the file is migrated to TypeScript, just to make it consistent with the rest of the functions

Reply:
Lets stick with the current solution